src/twisted/trial/_dist/test/test_workertrial.py

```diff
@@ -102,13 +102,16 @@ def test_forwardCommand(self):
         client = FakeAMP()
         clientTransport = StringTransport()
         client.makeConnection(clientTransport)
-        client.callRemote(workercommands.Run, testCase="doesntexist")
+        client.callRemote(workercommands.Run, testCase=b"doesntexist")
         self.readStream = clientTransport.io
         self.readStream.seek(0, 0)
         main(self.fdopen)
         self.assertIn(
             "No module named 'doesntexist'", self.writeStream.getvalue())
 
+    if _PY3:
+        test_forwardCommand.skip = "Does not work on Python 3 (https://tm.tl/8944)"
```

Comment:
I don't have a lot of context for how critical this test is. Should we consider folding the follow-up issue into this branch? If not, can you weigh in on the impact of possibly shipping a version of Twisted that doesn't test this?